docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,161 @@
+This file must have the special structure: tests must be organized into fixtures. So, first you need to declare a fixture by using the [fixture](../test-api/test-code-structure.md#fixtures) function.
+
+You can optionally specify a start page URL for the fixture by using the [page](../test-api/test-code-structure.md#specifying-the-start-webpage) function.
+In this tutorial, you will create a test for the [http://testcafe.devexpress.com/example](http://testcafe.devexpress.com/example) sample page.
+
+Then create the [test](/test-api/test-code-structure.md#tests) function where you will further place test code.
+
+```js
+fixture `Getting Started`
+    .page('http://testcafe.devexpress.com/example');
+
+test('Test1', async t => {
+    // Test code
+});
+```
+
+## Running the Test
+
+You can simply run the test from a command shell, by calling a single command where you specify the [target browser](../using-testcafe/command-line-interface.md#browser-list) and [file path](../using-testcafe/command-line-interface.md#file-pathglob-pattern).
+
+```bash
+testcafe safari test1.js
+```
+
+TestCafe will automatically open the chosen browser and start the test execution within it.
+
+For more information on how to configure the test run, see [Command Line Interface](../using-testcafe/command-line-interface.md).
+
+## Viewing the Test Results
+
+While the test is running, TestCafe is gathering information about the test run and outputing the report right into a command shell.
+
+![Test Report](../../images/report.png)
+
+For more information, see [Reporters](../using-testcafe/common-concepts/reporters.md).
+
+## Writing Test Code
+
+### Performing Actions on the Page
+
+Every test should be capable of interacting with a page content. To perform user actions, TestCafe provides a number of [actions](../test-api/actions.md): `click`, `hover`, `typeText`, `setFilesToUpload`, etc.
+They can be called in a chain.
+
+The following fixture contains a simple test that types a developer name into a text editor and then clicks the Submit button.
+
+```js
+fixture `Getting Started`
+    .page('http://testcafe.devexpress.com/example');
+
+test('Test1', async t => {
+    await t
+        .typeText('#developer-name', 'John Smith')
+        .click('#submit-button');
+});
+```
+
+All test actions are implemented as async functions of the [test controller object](../test-api/test-code-structure.md#test-controller) `t`. This object is used to access test run API.
+To wait for actions to be complete, use the `await` keyword when calling these actions or action chains.
+
+### Observing Page State
+
+TestCafe allows you to observe the page state.
+For this purpose, it offers special functions: [Selector](../test-api/executing-client-code/index.md#selector-functions) to get direct access to DOM elements and [Client](test-api/executing-client-code/index.md#client-functions) to obtain arbitrary data from the client side.
```

Comment:
It's `ClientFunction`, not `Client`. Also 

> it offers special functions

I think it's better to say `special kinds of functions`, because we don't offer the functions themselves, but the way to construct them.